modules/statemachine/statemachinemodule_lua.inl

```diff
@@ -114,4 +114,15 @@ int printCurrentStateInfo(lua_State* L) {
     return 1;
 }
 
+int saveToDotFile(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, { 1, 2 }, "lua::saveToDotFile");
+
```

Comment:
In the rewrite I didn't have any empty lines between the check arguments and the extraction line, but I'm good eitherway